algo-builder-tester/test/algob-tester.ts

```diff
@@ -6,10 +6,10 @@ import { types } from "@algo-builder/web";
 
 const DESCRIPTION = `
 TST1, TST2: Test Global.group_size()
-`
+`;
 const adminBalance = BigInt(1e8);
 const fee = 1e3;
-describe.skip('algob-tester', function () {
+describe.only("algob-tester", function () {
```

Comment:
drop `only`?

Reply:
Yes. Will drop all in new commit.

Reply:
fixed by https://github.com/Artcoin-Network/artificial-algorand-contract/commit/3c6576d23b9279e66b000a6c03152f7e35622792